src/Database/TigerBeetle/Raw/Client.hs

```diff
@@ -182,20 +192,25 @@ finalizeClient state = do
 submitRequest
   :: ClientState
   -> TBOperation
-  -> ByteString  -- ^ Request data
+  -> ByteString
+  -- ^ Request data
   -> IO (Either RequestError TBResponse)
 submitRequest state operation reqData = do
   -- Scaffold request state
-  res <- atomically $ readTVar state.csIsShutdown >>= \case
-    True -> pure (Left ClientShutdownDuringRequest)
-    False -> Right <$> provisionRequestContext state
+  res <-
+    atomically $
+      readTVar state.csIsShutdown >>= \case
+        True -> pure (Left ClientShutdownDuringRequest)
+        False -> Right <$> provisionRequestContext state
 
   case res of
     Left e -> pure $ Left e
     Right context -> withPacketPtrs reqData \packetPtr contextPtr (dataPtr, dataSize) -> do
       -- TODO: confirm that this is the correct way to assign a context id to a packet
       poke contextPtr context.contextId
-      let packet = TBPacket
+      let
+        packet =
+          TBPacket
```

Comment:
This is a bit ugly but an unfortunate effect of the let style I guess? Is there a way to prefer:
```
let foo = ...
      bar = ....
````
Without breaking the lines? Its unfortunate to have a whole line for one keyword.

Reply:
I was able to find a config to control this thankfully